src/modals/ManagePool/Tasks.tsx

```diff
@@ -125,28 +121,6 @@ export const Tasks = forwardRef(({ setSection, setTask }: any, ref: any) => {
               </button>
             </>
           )}
-          {isMember() && !isDepositor() && active?.isGreaterThan(0) && (
```

Comment:
Seems like maybe commenting out instead of removing would make it easier to restore this in the future